packages/api/src/schema/api-schema.yaml

```diff
@@ -1323,8 +1286,25 @@ components:
               $ref: "#/components/schemas/task/properties/params/properties/upload"
             export:
               $ref: "#/components/schemas/export-task-params"
-            transcode:
+            export-data:
```

Comment:
Any reason to use kebab-case here? If not, would prefer to keep the camelCase almost-consistency in the API today

Reply:
Updated in baf2406b29246e9e5db495bd099522d5e128a7c3

Though, I think we don't have any consistency here. Maybe we should think about cleaning up this whole file/api at some point.